cluster/gce/gci/configure.sh

```diff
@@ -682,6 +750,7 @@ set-broken-motd
 KUBE_HOME="/home/kubernetes"
 KUBE_BIN="${KUBE_HOME}/bin"
 
+ensure-docker
```

Comment:
this doesn't work for cos OS, at least "apt-get" doesn't work for cos OS

Reply:
fixed to add os-specific installs (supporting ubuntu for now), the rest of the function checks for docker installed, and sets config to overlay2 to work with COS-93.